### PR TITLE
Bumps version to 0.2.0

### PR DESCRIPTION
**Pull request by Davidslv**

### CODE REVIEW

File: lib/govuk_security_audit/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module GovukSecurityAudit
-  VERSION = "0.1.2"
+  VERSION = "0.1.3"
```
Comment (benlovell) [inline lib/govuk_security_audit/version.rb]: I'd argue that semantically this should be 0.2.2 as it adds a backwards compatible feature, not a bug fix.

**Comment by Davidslv** [comment]
@benlovell I've addressed your comment, thank you

**Comment by tijmenb** [comment]
This should actually be `0.2.0`, because it's the first in the `0.2.X` series :)

**Comment by benlovell** [comment]
@tijmenb Yep, you're right. My brain is mush. :smile:

**Comment by Davidslv** [comment]
@tijmenb @benlovell updated.

**Pull request merged**